packages/core/src/app/shipping/ConsignmentLineItemSummary.tsx

```diff
@@ -0,0 +1,33 @@
+import { PhysicalItem } from '@bigcommerce/checkout-sdk';
+import React, { FunctionComponent, memo } from 'react';
+
+import { MultiShippingTableItemWithType } from './MultishippingV2Type';
+
+export interface ConsignmentLineItemSummaryProps {
+    lineItems: MultiShippingTableItemWithType[] | PhysicalItem[]
+}
+
+const ConsignmentLineItemSummary: FunctionComponent<ConsignmentLineItemSummaryProps> = ({
```

Comment:
I suppose we can name it better since this is going to be used in allocate items modal as well. Something like `ConsignmentLineItemDetail`